pyproject.toml

```diff
@@ -5,7 +5,17 @@ description = "Add your description here"
 readme = "README.md"
 requires-python = ">=3.12"
 dependencies = [
-    "mcp[cli]>=1.5.0",
+    "mcp[cli]>=1.9.4",
+]
+
+[project.optional-dependencies]
+dev = [
```

Comment:
Hm, this file has both optional-dependencies and dependency-groups, and they have different values. Not sure why the difference? Do we need both?
(I found this issue about it here:
https://github.com/astral-sh/uv/issues/9011)

Reply:
thanks for link, updated to use dependency-group